.kokoro/release.cfg

```diff
@@ -31,3 +31,12 @@ env_vars: {
   key: "SECRET_MANAGER_KEYS"
   value: "releasetool-publish-reporter-app,releasetool-publish-reporter-googleapis-installation,releasetool-publish-reporter-pem,docuploader_service_account"
 }
+
+# Store the packages uploaded to rubygems.org, which
+# we can later use to generate SBOMs and attestations.
+action {
+ define_artifacts {
+   regex: "github/google-cloud-ruby/**/pkg/*.gem"
```

Comment:
I think you can use `github/google-cloud-ruby/*/pkg/*.gem`. There should always be only one directory level for a gem. Might allow us to avoid an edge case where a gem archive shows up in, for example, a test directory.